examples/pipeline_wavernn/main.py

```diff
@@ -0,0 +1,402 @@
+        default=4,
+        type=int,
+        metavar="N",
+        help="number of data loading workers",
+    )
+    parser.add_argument(
+        "--checkpoint",
+        default="",
+        type=str,
+        metavar="PATH",
+        help="path to latest checkpoint",
+    )
+    parser.add_argument(
+        "--epochs",
+        default=8000,
+        type=int,
+        metavar="N",
+        help="number of total epochs to run",
+    )
+    parser.add_argument(
+        "--start-epoch", default=0, type=int, metavar="N", help="manual epoch number"
+    )
+    parser.add_argument(
+        "--print-freq",
+        default=10,
+        type=int,
+        metavar="N",
+        help="print frequency in epochs",
+    )
+    parser.add_argument(
+        "--batch-size", default=256, type=int, metavar="N", help="mini-batch size"
+    )
+    parser.add_argument(
+        "--learning-rate", default=1e-4, type=float, metavar="LR", help="learning rate",
+    )
+    parser.add_argument("--clip-grad", metavar="NORM", type=float, default=4.0)
+    parser.add_argument(
+        "--mulaw",
+        default=True,
+        action="store_true",
+        help="if used, waveform is mulaw encoded",
+    )
+    parser.add_argument(
+        "--upsample-scales",
+        default=[5, 5, 11],
+        type=List[int],
+        help="the list of upsample scales",
+    )
+    parser.add_argument(
+        "--n-bits", default=8, type=int, help="the bits of output waveform",
+    )
+    parser.add_argument(
+        "--sample-rate",
+        default=22050,
+        type=int,
+        help="the rate of audio dimensions (samples per second)",
+    )
+    parser.add_argument(
+        "--hop-length",
+        default=275,
+        type=int,
+        help="the number of samples between the starts of consecutive frames",
+    )
+    parser.add_argument(
+        "--win-length", default=1100, type=int, help="the length of the STFT window",
+    )
+    parser.add_argument(
+        "--f-min", default=40.0, type=float, help="the minimum frequency",
+    )
+    parser.add_argument(
+        "--min-level-db",
+        default=-100,
+        type=float,
+        help="the minimum db value for spectrogam normalization",
+    )
+    parser.add_argument(
+        "--n-res-block", default=10, type=int, help="the number of ResBlock in stack",
+    )
+    parser.add_argument(
+        "--n-rnn", default=512, type=int, help="the dimension of RNN layer",
+    )
+    parser.add_argument(
+        "--n-fc", default=512, type=int, help="the dimension of fully connected layer",
+    )
+    parser.add_argument(
+        "--kernel-size",
+        default=5,
+        type=int,
+        help="the number of kernel size in the first Conv1d layer",
+    )
+    parser.add_argument(
+        "--n-freq", default=80, type=int, help="the number of spectrogram bins to use",
+    )
+    parser.add_argument(
+        "--n-hidden", default=128, type=int, help="the number of hidden dimensions",
```

Comment:
cc [comment](https://github.com/pytorch/audio/pull/735/files#r456481205)

Reply:
It has been changed to:
```
parser.add_argument(
        "--n-hidden-resblock", default=128, type=int, help="the number of hidden dimensions of resblock",
    )
```